inference.py

```diff
@@ -309,9 +267,12 @@ def main(params: dict):
         with tqdm(list_img, desc='image list', position=0) as _tqdm:
             for info in _tqdm:
                 img_name = Path(info['tif']).name
-                local_gpkg = info['gpkg']
+                local_gpkg = info['gpkg'] if 'gpkg' in info.keys() else None
                 if local_gpkg:
                     local_gpkg = Path(local_gpkg)
+                    gpkg_name = Path(local_gpkg).stem
+                else:
+                    gpkg_name = None
```

Comment:
Related to previous comment. I like one-liners for things like this:
`gpkg_name = local_gpkg.stem is local_gpkg else None`

Reply:
better code syntax, I just need to check if the truth evaluation works with Path.